app/dashboard/views.py

```diff
@@ -264,7 +264,7 @@ def new_interest(request, bounty_id):
             status=401)
 
     try:
-        Interest.objects.get(profile_id=profile_id, bounty=bounty)
+        Interest.objects.exclude(status__in=Interest.INACTIVE_STATUSES).get(profile_id=profile_id, bounty=bounty)
```

Comment:
@SaptakS can you help me with  this one? I think create_new_interest_helper need to be updated so if the user stop work and the start again this update the entry and not create a new one. Does that make sense?

Reply:
@octavioamu can't we use `active()` here ?

Reply:
@SaptakS I resolved this by mistake but I need your helpo about that BE logic.